vowpalwabbit/array_parameters_dense.h

```diff
@@ -114,6 +114,49 @@ class dense_parameters
     for (iterator iter = begin(); iter != end(); ++iter) (&(*iter))[offset] = 0;
   }
 
+  void copy_offsets(const size_t from, const size_t to, const size_t params_per_problem)
```

Comment:
Generally speaking we don't use const for value args across the codebase so I'd prefer without, but if you feel strongly thats okay

Reply:
I wanted to communicate that those vars don't get used as buffers or mutate around in this function, since the fn has already a lot going on.